### PR TITLE
Specs cleanup

### PR DESCRIPTION
**Pull request by qaseio**
Changes:
- Removed examples (they are not necessary, we can use swagger-codegen locally)
- Removed webhooks markdown files (will be moved to developers.qase.io as a guide)
- Tags are in Uppercase now
- Removed description block (moved to guide in developers.qase.io)
- CodeOrHash and IdOrHash are split into Id/Hash/Code

### CODE REVIEW

File: paths/case.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ delete:
 
 patch:
   operationId: update-case
-  tags: [ cases ]
+  tags: [ Сases ]
```
Comment (abelharisov) [inline paths/case.yaml:37]: C is cyrillic here

File: paths/cases.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 get:
   operationId: get-cases
-  tags: [ cases ]
+  tags: [ Сases ]
```
Comment (abelharisov) [inline paths/cases.yaml:3]: And in this file too

File: paths/custom_field.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 get:
   operationId: get-custom-field
-  tags: [ custom-fields ]
+  tags: [ Сustom Fields ]
```
Comment (abelharisov) [inline paths/custom_field.yaml:3]: Here and below C is cyrillic too

File: paths/users.yaml [hunk 4]
```diff
@@ -1,16 +0,0 @@
-get:
```
Comment (abelharisov) [inline paths/users.yaml:1]: Why we are removing it? Should we also remove this API from our backend?

File: src.yaml [hunk 5]
```diff
@@ -2,9 +2,6 @@ openapi: '3.0.0'
 
 info:
   title: Qase.io API
-  description:
```
Comment (abelharisov) [inline src.yaml:5]: Some description should present
image

File: paths/attachment.yaml [hunk 6]
```diff
@@ -44,7 +44,7 @@ post:
     If there is no free space left in your team account,
     you will receive an error with code 507 - Insufficient Storage.
   parameters:
-    - $ref: '../parameters/CodeOrHash.yaml'
+    - $ref: '../parameters/Hash.yaml'
```
Comment (abelharisov) [inline paths/attachment.yaml:47]: It accepts Project code here. Not hash
That's why CodeOrHash was introduced

Comment (abelharisov) [reply paths/attachment.yaml:47]: Maybe we can reflect it in the description?

**Review by abelharisov — APPROVED** [review]

**Comment by kamilsk** [comment]
> Removed examples (they are not necessary, we can use swagger-codegen locally)

please restore stubs (`*.http` in examples dir)

**Comment by kamilsk** [comment]
> Removed webhooks markdown files (will be moved to developers.qase.io as a guide)

> Removed webhooks markdown files (will be moved to developers.qase.io as a guide)

please move it first, then delete

**Comment by kamilsk** [comment]
> Tags are in Uppercase now

what is the motivation to do that?

**Comment by kamilsk** [comment]
> Removed description block (moved to guide in developers.qase.io)

what problem are we solving?

File: package.json [hunk 7]
```diff
@@ -24,8 +24,5 @@
     "@stoplight/spectral-cli": "^6.1.0",
     "ansi-regex": "^6.0.1",
     "redoc-cli": "^0.13.0"
-  },
-  "directories": {
-    "example": "examples"
```
Comment (kamilsk) [inline package.json:29]: with smokes this dir is useful

File: paths/attachment.yaml [hunk 8]
```diff
@@ -1,11 +1,11 @@
 get:
   operationId: get-attachment
-  tags: [ attachments ]
+  tags: [ Attachments ]
   summary: Get attachment by Hash.
   description: |
     This method allows to retrieve attachment by Hash.
   parameters:
-    - $ref: '../parameters/CodeOrHash.yaml'
+    - $ref: '../parameters/Hash.yaml'
```
Comment (kamilsk) [inline paths/attachment.yaml:8]: code != hash, it doesn't matter that they have the same type under the hood

File: paths/result.yaml [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
   operationId: get-result
-  tags: [ results ]
+  tags: [ Results ]
   summary: Get test run result by code.
   description: |
     This method allows to retrieve a specific test run result by Hash.
   parameters:
     - $ref: '../parameters/project/Code.yaml'
-    - $ref: '../parameters/IdOrHash.yaml'
+    - $ref: '../parameters/Hash.yaml'
```
Comment (kamilsk) [inline paths/result.yaml:9]: now, id (integer) is not acceptable here. why?

File: paths/user.yaml [hunk 10]
```diff
@@ -1,15 +0,0 @@
-get:
```
Comment (kamilsk) [inline paths/user.yaml:1]: what is the reason? we had it before, and we have public methods for it

File: src.yaml (hunk 5 above)
Comment (kamilsk) [reply src.yaml:5]: please provide a simple "one-liner" instead of the markdown

**Review by kamilsk — CHANGES_REQUESTED** [review]

**Comment by kamilsk** [comment]
most of your changes were implemented and in master, except tags, users (hide instead deletion)

**Pull request closed without merging**